src/reducers/fetch.js

```diff
@@ -45,7 +45,14 @@ export const success = (state, action) => {
     if(Array.isArray(responseData)) {
 
       // Merge new raw data with existing raw data
-      const normalizedData = normalize(action.uidField, Immutable.fromJS(responseData))
+      let normalizedData = normalize(action.uidField, Immutable.fromJS(responseData)).map((item, uid) => {
+        // If an item is already saved merge the two items
+        if (state.getIn(['raw', uid.toString()])) {
+          return state.getIn(['raw', uid.toString()]).merge(item)
+        }
+        return item
+      })
+
       map.set('raw', state.get('raw').merge(normalizedData))
```

Comment:
If we switch the `merge` here to `mergeDeep` we shouldn't need to map over the values above.